internal/pkg/cli/pipeline_init.go

```diff
@@ -326,42 +402,86 @@ func (o *initPipelineOpts) parseBitbucketRepoDetails() error {
 	o.repoName = repoDetails.name
 	o.repoOwner = repoDetails.owner
 
-	if o.repoBranch == "" {
-		o.repoBranch = defaultBBBranch
-	}
 	return nil
 }
 
 func (o *initPipelineOpts) selectURL() error {
-	// Fetches and parses all remote repositories.
-	err := o.runner.Run("git", []string{"remote", "-v"}, exec.Stdout(&o.buffer))
-	if err != nil {
-		return fmt.Errorf("get remote repository info: %w; make sure you have installed Git and are in a Git repository", err)
-	}
-	urls, err := o.parseGitRemoteResult(strings.TrimSpace(o.buffer.String()))
+	URLsAndRemoteNames, err := o.fetchAndParseURLs()
 	if err != nil {
 		return err
 	}
-	o.buffer.Reset()
+
+	var formattedRemoteNamesAndURLs []string
+	for _, repo := range URLsAndRemoteNames {
+		formattedRemoteNamesAndURLs = append(formattedRemoteNamesAndURLs, fmt.Sprintf(fmtRepoSelection, repo.remoteName, repo.remoteURL))
+	}
+	if len(formattedRemoteNamesAndURLs) == 1 {
+		log.Infof("Only one git repository detected. Your pipeline will follow '%s'. You may make changes in the generated pipeline manifest before deployment.\n", color.HighlightUserInput(formattedRemoteNamesAndURLs[0]))
```

Comment:
nit: since the URL that the pipeline will follow is really `repo.remoteURL` (without the prepending colon and remote name)- what do you think of 
```suggestion
		log.Infof("Only one git repository detected. Your pipeline will follow '%s'. You may make changes in the generated pipeline manifest before deployment.\n", color.HighlightUserInput(repo[0].remoteURL))
```

Reply:
great idea!